openquake/hazardlib/gsim/lanzano_2016.py

```diff
@@ -0,0 +1,370 @@
+	1.600	-0.409	0.946	-0.045	-1.238	0.150	-1.214	0.046	-1.128	0.323	-1.278	0.072	5.033	0.000	0.070	0.000	0.000	0.085	0.303	0.115	0.099	0.297	0.313
+	1.800	-0.486	0.977	-0.039	-1.218	0.141	-1.239	0.066	-1.103	0.313	-1.315	0.014	4.737	-0.001	0.067	0.000	0.000	0.080	0.303	0.119	0.100	0.299	0.315
+	2.000	-0.554	0.997	-0.037	-1.189	0.138	-1.263	0.069	-1.100	0.282	-1.345	0.057	4.241	-0.010	0.056	0.000	0.000	0.081	0.308	0.117	0.099	0.298	0.314
+	2.500	-0.742	1.034	-0.027	-1.164	0.151	-1.326	0.045	-1.072	0.299	-1.385	0.060	4.126	0.040	0.054	0.000	0.000	0.085	0.327	0.115	0.100	0.301	0.317
+	3.000	-0.881	1.057	-0.019	-1.152	0.165	-1.378	0.018	-1.020	0.339	-1.449	0.084	4.170	0.072	0.045	0.000	0.000	0.089	0.325	0.114	0.101	0.304	0.320
+	4.000	-1.084	1.134	0.019	-1.101	0.244	-1.488	-0.153	-0.971	0.414	-1.619	-0.119	4.454	0.073	0.019	0.000	0.000	0.096	0.322	0.131	0.113	0.298	0.318
+	pga		0.071	0.603	-0.019	-1.895	0.286	-0.926	0.035	-1.838	0.511	-2.256	0.455	6.701	0.035	0.181	0.000	0.000	0.050	0.203	-0.060	0.106	0.318	0.336
+	pgv		-1.142	0.767	-0.005	-1.623	0.230	-1.037	-0.054	-1.596	0.379	-1.741	0.348	5.904	0.022	0.144	0.000	0.000	0.085	0.260	0.037	0.096	0.288	0.304
+	""")
+
+
+class LanzanoEtAl2016_Rhypo(LanzanoEtAl2016_RJB):
+    """
+    Implements GMPE developed by G.Lanzano, M. D'Amico, C.Felicetta, R.Puglia, 
+    L.Luzi, F.Pacor, D.Bindi and published as "Ground-Motion Prediction Equations
+    for Region-Specific Probabilistic Seismic-Hazard Analysis",
+    Bull Seismol. Soc. Am., DOI 10.1785/0120150096
+    SA are given up to 4 s.
+    The regressions are developed considering the geometrical mean of the
+    as-recorded horizontal components
+    """
+    #: Supported tectonic region type is 'active shallow crust' because the
+    #: equations have been derived from data from Italian database ITACA, as
+    #: explained in the 'Introduction'.
+    DEFINED_FOR_TECTONIC_REGION_TYPE = const.TRT.ACTIVE_SHALLOW_CRUST
+
+    #: Set of :mod:`intensity measure types <openquake.hazardlib.imt>`
+    #: this GSIM can calculate. A set should contain classes from module
+    #: :mod:`openquake.hazardlib.imt`.
+    DEFINED_FOR_INTENSITY_MEASURE_TYPES = set([
+        PGA,
+        PGV,
+        SA
+    ])
+
+    #: Supported intensity measure component is the geometric mean of two
+    #: horizontal components
+    DEFINED_FOR_INTENSITY_MEASURE_COMPONENT = const.IMC.AVERAGE_HORIZONTAL
+
+    #: Supported standard deviation types are inter-event, intra-event
+    #: and total, page 1904
+    DEFINED_FOR_STANDARD_DEVIATION_TYPES = set([
+        const.StdDev.TOTAL,
+        const.StdDev.INTER_EVENT,
+        const.StdDev.INTRA_EVENT
+    ])
+
+    #: Required site parameter is only Vs30
+    REQUIRES_SITES_PARAMETERS = {'vs30','lon','lat','bas'}
+
+    #: Required rupture parameters are magnitude and rake.
+    REQUIRES_RUPTURE_PARAMETERS = {'rake', 'mag'}
+
+    #: Required distance measure is R Hypocentral Distance.

```

Comment:
Since you are extending `LanzanoEtAl2016_RJB` you just need to add here the variables that differ from the original class i.e. the distance parameter.